src/renderer/constants/keystore.ts

```diff
@@ -0,0 +1 @@
+export const KEYSTORE_DEFAULT_DIRECTORY = "./keystore";
```

Comment:
should be user home directory, check some electron libs to determine best place across platforms